src/pymovements/gaze/experiment.py

```diff
@@ -97,7 +98,65 @@ def __init__(
             distance_cm: float | None = None,
             origin: str = 'lower left',
             sampling_rate: float | None = None,
+            eyetracker: EyeTracker | None = None,
     ):
+        """Initialize Experiment.
```

Comment:
Starting #638 we don't allow for `__init__()` method docstrings anymore.
Please remove this docstring entirely and merge it into the docstring above (most of the text has been moved there already).
The sentence `Initialize Experiment.` wont' be necessary anymore.